mountpoint-s3/src/inode.rs

```diff
@@ -839,6 +837,57 @@ pub enum InodeError {
     InodeNotReadableWhileWriting(InodeNo),
 }
 
+mod prefix {
```

Comment:
Not sure if we should define prefix in a separate file and assign the related path in `lib.rs` ?

Reply:
Moved to prefix.rs - but happy to change if there is a better location.